### PR TITLE
feat[w1.3]: modified page for unpublished feedbacks

### PR DESCRIPTION
**Pull request by jelenazeljko55**

### CODE REVIEW

**Comment by jelenazeljko55** [comment]
[(3) W1.3 : As a system administrator I want to publish feedback so that I could share positive experiences with other users.](https://trello.com/c/CaAvA5FG/3-3-w13-as-a-system-administrator-i-want-to-publish-feedback-so-that-i-could-share-positive-experiences-with-other-users)

**Review by anciz — DISMISSED** [review]

**Review by milana-todorovic — CHANGES_REQUESTED** [review]
Could you change 'Hospital' on the navbar to 'HOSPITAL' so it matches the other pages? The white rectangle on the background on this page is also somewhat odd, I'm not sure if it was on purpose or accidental.

File: HealthcareSystem/PatientWebApp/wwwroot/js/unpublished_feedback.js [hunk 1]
```diff
@@ -1,21 +1,68 @@
...
+		type: "GET",
+		dataType: 'json',
+		processData: false,
+		contentType: false,
+		success: function (data) {
+			let length = data.length;
+			for (let i = 0; i < length; i++) {
+				addCommentTable(data[i]);
+			}
+		},
```
Comment (anciz) [inline HealthcareSystem/PatientWebApp/wwwroot/js/unpublished_feedback.js]: This is not clear enough. Try:

```sh
		success: function (data) {
			for (let i = 0; i < data.length; i++) {
				 addCommentTable(data[i]);
			}
		},
```

File: HealthcareSystem/PatientWebApp/wwwroot/js/unpublished_feedback.js [hunk 2]
```diff
@@ -1,21 +1,68 @@
...
+		trElement.append(tdElement);
+	}
+	else {
+		let tdElement = $('<td style="color:red;"> Publishing disabled </td>');
+		trElement.append(tdElement);	
+	}
+	divElement.append(trElement);
+	$('div#div_comments').append(divElement);
+
+}
```
Comment (anciz) [inline HealthcareSystem/PatientWebApp/wwwroot/js/unpublished_feedback.js]: What is 'c'?
Give more descriptive name.

**Review by anciz — CHANGES_REQUESTED** [review]

**Review by anciz — APPROVED** [review]
Well done! 
You changed everything that was requested.

**Review by milana-todorovic — APPROVED** [review]
Looks nice!

**Pull request merged**